starlette/routing.py

```diff
@@ -231,8 +231,6 @@ def __init__(
             # Endpoint is a class. Treat it as ASGI.
             self.app = endpoint
 
-        regex = "^" + path + "$"
-        regex = re.sub("{([a-zA-Z_][a-zA-Z0-9_]*)}", r"(?P<\1>[^/]+)", regex)
```

Comment:
Any idea where these lines have been left over from? They don't make sense right now, but might be a good idea to understand how they came to be here?

Reply:
This [commit](https://github.com/encode/starlette/commit/de71b8594807dae907eba636f01dc13fdb355f9d#diff-34fba745b50527bfb4245d02afd59246L122) removed the pattern in `Route`, but forgot `WebSocketRoute`.